rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,424 @@
+        globals()['background_'+uid] = Dropdown(description='', options= bgcolors_3d, value=bgcolors_3d[1])
+        wgListBox.append(Box([Label(value='color'),globals()['colorScheme_'+uid]], layout=itemLayout))
+        wgListBox.append(Box([Label(value='background'),globals()['background_'+uid]], layout=itemLayout))
+        globals()['rdkit_wg_dict'].update({globals()['colorScheme_'+uid]._model_id:uid})
+        globals()['rdkit_wg_dict'].update({globals()['background_'+uid]._model_id:uid})
+        globals()['colorScheme_'+uid].observe(handle_change, names='value')
+        globals()['background_'+uid].observe(handle_change, names='value')
+    
+    
+    # labels
+    if labelPanel is True:
+        globals()['confLabel_'+uid] = Checkbox(description='confLabel', value=False)
+        globals()['atomLabel_'+uid] = Checkbox(description='atomLabel', value=False)
+        
+        cbLabel=Box([Label(value=''),
+                     HBox([globals()['confLabel_'+uid], 
+                           globals()['atomLabel_'+uid]])
+                    ], layout=itemLayout)
+        
+        wgListBox.append(cbLabel)
+        
+        globals()['rdkit_wg_dict'].update({globals()['confLabel_'+uid]._model_id:uid})
+        globals()['rdkit_wg_dict'].update({globals()['atomLabel_'+uid]._model_id:uid})
+        globals()['confLabel_'+uid].observe(handle_change, names='value')
+        globals()['atomLabel_'+uid].observe(handle_change, names='value')
+    
+    
+    # buttons
+    globals()['start_'+uid] = Button(description="Start!", button_style='success')
+    globals()['zoomTo_'+uid] = Button(description="zoomTo", button_style='success')
+    buttons=Box([Label(value=''),HBox([globals()['zoomTo_'+uid], globals()['start_'+uid]])],layout=itemLayout)
+    wgListBox.append(buttons)
+    
+    globals()['rdkit_wg_dict'].update({globals()['start_'+uid]._model_id:uid})
+    globals()['rdkit_wg_dict'].update({globals()['zoomTo_'+uid]._model_id:uid})
+    globals()['start_'+uid].on_click(handle_start_button)
+    globals()['zoomTo_'+uid].on_click(handle_zoomTo_button)
+    
+    
+    # left panel (container holding table for viewer)
+    size = (435, 485)
+    viewerLayout=Layout(width=str(size[0]+5)+'px', height=str(size[1]+5)+'px', border='solid')
+    wgLeftWidget = HTML('''<table><tr><td id="%s"></td></tr></table>'''%uid, layout=viewerLayout)
+    
+    # right panel
+    wgRightBox=VBox(wgListBox, layout=Layout(border='solid', width='45%'))
+    
+    # Combining left and right panel
+    all_wg = HBox([wgLeftWidget, wgRightBox])
+    
+    # displaying everything
+    display(all_wg)
+    
+    
+    # inserting 3DMol.js viewer in existing container (table)
+    globals()['view_'+uid] = py3Dmol.view(width=size[0],height=size[1])
+    globals()['view_'+uid].setBackgroundColor('0x000000')
+    globals()['view_'+uid].zoomTo()
+    
+    display(globals()['view_'+uid].insert(uid))
+    
```

Comment:
All those codes are cloned in the current code base. Many thanks for providing this. It makes life easy. 
This line `globals()['confId_'+uid].options = molViewState.allConfIds()` changed to `globals()['confId_'+uid].options = molViewState.allConfIds`

I have changed the object name. So, `globals()['molTupleId_'+uid].value` changed to `globals()['molId_'+uid].value`

Inside the class 
return list(range(nconfIDs)) changed to return list(range(nconfIds))

Those codes are working fine. Many thanks for that. Have a great day!